assets/src/util/sentry.tsx

```diff
@@ -11,12 +13,11 @@ const initSentry = (appString: string) => {
 
   // Note: passing an empty string as the DSN sets up a "no-op SDK" that captures errors and lets you call its methods,
   // but does not actually log anything to the Sentry service.
-  Sentry.init({
-    dsn: sentryDsn && isRealScreen() ? sentryDsn : "",
-    environment: env,
-  });
 
-  Sentry.captureMessage(`Sentry intialized for app: ${appString}`);
+  if (sentryDsn && isRealScreen()) {
+    Raven.config(sentryDsn, {environment: env}).install();
+    Raven.captureMessage(`Sentry intialized for app: ${appString}`, {level: "info"});
+  }
 };
 
 export default initSentry;
```

Comment:
Do you think it might also be useful to export a function (or functions) that wraps `Raven.captureMessage` so that if/when we need to change what gets called (e.g. switching back to `Sentry.captureMessage`, or something else), the change only needs to happen in one place?

Like:
```tsx
// https://docs.sentry.io/clients/javascript/usage/#raven-js-additional-context
type LogLevel = "info" | "warning" | "error";

const log = (message: string, level: LogLevel) => {
  Raven.captureMessage(message, {level});
};
const info = (message: string) => log(message, "info");
const warn = (message: string) => log(message, "warning");
const error = (message: string) => log(message, "error");

export default initSentry; // (unchanged)
export {info, warn, error};

//// in another file ////
import * as Logger from "Util/sentry";

Logger.info("something happened");
Logger.error("something bad happened");
```

Reply:
Oh sure, thanks! Tested the update, and it works nicely.